packages/examples/.gitignore

```diff
@@ -0,0 +1,176 @@
+# Based on https://raw.githubusercontent.com/github/gitignore/main/Node.gitignore
```

Comment:
We already have a .gitignore on the root of the project

Is all this just for the bun ignore?

Reply:
Addressed by 22003d7f5c3882357ca0ac9075cc912be249dabb: chore(examples): rm accidental inclusion of gitignore